src/interactive.js

```diff
@@ -0,0 +1,40 @@
+import Methods from './utils.js';
+
+export default class Interactive {
+static toggleCompleted = (id, curStatus) => {
+  const todoList = Methods.getLocalStorageData();
+  todoList[id].completed = curStatus;
+  Methods.setLocalStorageData(todoList);
+  Methods.showTodoItems();
+}
+
+static addCheckEvent = () => {
+  document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('click', () => {
+    // event.preventDefault();
```

Comment:
- [ ] Good Job @V-Blaze, Your code is looking great already, yet it's not a good idea to leave a code comment, as this will make your codebase messy.

Reply:
Thanks. comment removed.